river/sync.go

```diff
@@ -50,7 +51,19 @@ func (h *eventHandler) OnRotate(e *replication.RotateEvent) error {
 	return h.r.ctx.Err()
 }
 
-func (h *eventHandler) OnDDL(nextPos mysql.Position, _ *replication.QueryEvent) error {
+func (h *eventHandler) OnDDL(nextPos mysql.Position, e *replication.QueryEvent) error {
+	mb := checkRenameTable(e)
```

Comment:
seem this is only for rename table DDL, not for others like add/del columns.

Reply:
also works for  alter table ddl
maybe need to change the func name ? :-)

Reply:
`checkRenameTable `  ->  `getTableFromDDL `